### ISSUE TITLE
Tab navigation in the login activity

### ISSUE THREAD

**Issue by hirro** [hirro/m-worktajm#10]
As a user, I should be able to login using keyboard only, so that I don't have to switch between keyboard and touch. 

Note: User must press username to start the username password login type. The preferred login type is facebook or google+, those must not be covered by the keyboard. I.e, username does not have focus per default.
